lua/pixelui/core/cl_images.lua

```diff
@@ -56,18 +57,17 @@ end
 function PIXEL.GetImage(url, callback, matSettings)
     url = string.gsub(url, "https://", "")
     url = string.gsub(url, "http://", "")
-    local urlExploded = string.Explode("/", url)
-    local fileName = urlExploded[#urlExploded]
-    local urlNoFile = string.sub(url, 1, url:find("/[^/]+$") - 1)
+    local fileName = url:match("[^/]+$")
+    local urlWithoutFileName = string.sub(url, 1, url:find("/[^/]+$") - 1)
 
-    local dirPath = "pixel/" .. urlNoFile
+    local dirPath = PIXEL.DownloadPath .. "/" .. urlWithoutFileName
```

Comment:
Should should require the user to postfix their download path with "/" as opposed to concatenating a slash every time it is used.